### ISSUE TITLE
SSL Configuration stuck - Route 53 domain

### ISSUE THREAD

**Issue by omri-shilton** [aws-amplify/amplify-hosting#3637]
### Before opening, please confirm:

- [X] I have checked the [custom domains troubleshooting guide](https://docs.aws.amazon.com/amplify/latest/userguide/custom-domain-troubleshoot-guide.html).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### App Id

dpm5giteybegs

### Region

eu-west-2

### Step

SSL Configuration

### Status

SSL configuration

### Issue or question

Im trying to set up another domain for my app. I have my domain hosted in Route53 in the same account and i can see that both the CNAMEs are configured perfectly.
Yet the domain registration is still stuck on SSL Configuration

**Comment by Jay2113** [comment]
Hi @omri-shilton 👋 , I observed that you have been able to successfully connect a custom domain to your Amplify app `dpm5giteybegs`. Are you still running into this issue?

**Comment by XPGeek** [comment]
Observed similar, but temporary, behavior given the following environment:

### Before opening, please confirm:

- [x]  I have checked the [custom domains troubleshooting guide](https://docs.aws.amazon.com/amplify/latest/userguide/custom-domain-troubleshoot-guide.html).

- [x]  I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).

- [x]  I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### App Id
dy6ewpr2d09ta

### Region
us-east-1

### Step
SSL Configuration

### Status
SSL configuration

### Issue or question

Custom domain registration for Amplify app. Amplify app created 2023-08-15 @ 12:51:46 (EST). Once application was in running state, a custom domain (xptech.help) was registered via Route53 with "Domain registration successful" sent at 2023-08-15 13:31:39. Amplify Domain management process fired after verifying DNS records were had "INSYNC" status. SSL Configuration step creates CNAME record within Route53 instantly upon step initiation, but appears to hang until ~14:05 (~30min) which could lead to erronous ticket creation. Followed troubleshooting steps to verify name servers matched in meantime and typed out most of this comment checking back and forth.

TLDR - Suggest UX improvement of message to user during "SSL Verification" indicating process could take ~30min/1hr/2hr for recently created Route53 domains if not a fluke/recurring issue.

Appreciate the team, really simple workflows and overall great experience!

**Comment by Jay2113** [comment]
Hi @XPGeek 👋 , thanks for reaching out and for sharing detailed feedback on the domain activation process with Amplify Hosting for newly created domains with Route53. I'll relay this feedback to the team internally on improving the console experience.
